src/dataframe/dataframe.jl

```diff
@@ -759,7 +792,8 @@ end
 ##
 ##############################################################################
 
-pool(a::AbstractVector) = compact(PooledDataArray(a))
+pool(a::AbstractVector) = compact(NominalArray(a))
+pool{T<:Nullable}(a::AbstractVector{T}) = compact(NullableNominalArray(a))
 
 function pool!(df::DataFrame, cname::@compat(Union{Integer, Symbol}))
```

Comment:
No idea, but you're probably right that it's an oversight. Care to make a PR to fix this on master too?